docs/UserScriptOverview.md

```diff
@@ -16,7 +17,7 @@ There are several repositories containing example Scripts and tools available fo
 * [jcooper-korg's scripts](https://github.com/jcooper-korg/talon_user)
 * [lexjacobs’ scripts](https://github.com/lexjacobs/talon_user)
 
-In the future, Talon will ship with a standard set of Voice Commands and language rules into which other systems can more consistently be built. For now, however, it is largely left to the individual user to download or make the Voice Commands and Scripts he wishes to use.
+In the future, Talon will ship with a standard set of Voice Commands and language rules into which other systems can more consistently be built. For now, however, there is an [unofficial community set of scripts](https://github.com/dwiel/talon_community) which can be used as a starting point (pull requests welcome). There are also a large number of public user scripts listed above. It is largely left to the individual user to download or make the Voice Commands and Scripts they wish to use.
```

Comment:
I do not agree with calling out any particular source of community scripts, except official sources. I think it's inappropriate to place your own scripts with a privileged position within the documentation, and the "(pull requests welcome)" is off topic. Just put in a link like everyone else's repository.

I understand the desire to change "he" to "they", but I chose that language of the English generic "he" to avoid the unnecessary pluralization that "they". If it really bothers you, then we can just rephrase to something like "For now, however, users are largely encouraged to download or make the Voice Commands and Scripts they wish to use."

Reply:
I understand that position. I called out this particular source of community scripts because as far as i'm aware, it is the only repository whose goal is providing people with a comprehensive starting point. All the others are personal repositories with a side benefit of being public. I can put it with the others and a note to that effect?

he/they is not a big deal to me, I just try to be sensitive to it if i see it, and changed it while i was in here.